src/main/webapp/WEB-INF/clinic/edit.html

```diff
@@ -48,6 +48,34 @@
 
 					/>
 				</div>
+				<div class="accordion" id="accordionExample">
+
+				<div class="accordion-item">
+					<h2 class="accordion-header" id="headingOne">
+					  <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapseOne" aria-expanded="false" aria-controls="collapseOne">
+						<th:block
+						th:text="${messages.get(#locale, 'clinic.heading.clinicConfiguration', 'Clinic Configuration')}"
+					/>					  </button>
```

Comment:
Looks faulty. Please make closing bracket in line with opening one, indent content and move closing button tag to new line